server/util/redact/redact_test.go

```diff
@@ -110,7 +110,7 @@ func TestRedactMetadata_StructuredCommandLine(t *testing.T) {
 		{"some_url", "https://token@foo.com", "https://foo.com"},
 		{"remote_default_exec_properties", "container-registry-username=SECRET_USERNAME", "container-registry-username=<REDACTED>"},
 		{"remote_default_exec_properties", "container-registry-password=SECRET_PASSWORD", "container-registry-password=<REDACTED>"},
-		{"host_platform", "@buildbuddy_toolchain//:platform", "@buildbuddy_toolchain//:platform"},
+		{"host_platform", "//platforms:local_config_platform", "//platforms:local_config_platform"},
```

Comment:
this test could be left as-is.

I hope that we could upstream some changes in the `//platforms` dir to `@buildbuddy_toolchain` eventually.

Reply:
👍